src/custom-element.js

```diff
@@ -976,6 +976,14 @@ function createBaseCustomElementClass(win) {
       return this.implementation_.createPlaceholderCallback();
     }
 
+    /**
+     * @return {?Element}
+     * @final @this {!Element}
+     */
+    createLoaderBrand() {
```

Comment:
`createBrandedLoader`? 😜

Reply:
you will get your chance for through bike-shedding soon :) probably going to `s/brand*/logo`